data/assets/examples/geojson/geojson_extruded_shaded_polygon.asset

```diff
@@ -7,7 +7,7 @@ local data = asset.resource({
   Name = "GeoJSON Example Africa",
   Type = "UrlSynchronization",
   Identifier = "geojson_example_polygon_extruded_africa",
-  Url = "http://liu-se.cdn.openspaceproject.com/files/examples/geojson/polygon_extruded_africa.geojson"
+  Url = "http://liu-se.cdn.openspaceproject.com/files/examples/geojson/polygon_extruded_africa.geojson",
```

Comment:
Why add these trailing commas?  (A bit everywhere)

since @alexanderbock approved the changes, I assume there is a reason for adding them?

Reply:
That is on me, previously we had specified secondsuntilResync = math.huge in these files as well. But we decided it was unnecessary to include it because it is not clear what it means and the default behavior already handles that on the c++ side. So this was just an oversight when removing that variable in those files

Reply:
Ah ok!  Please remove all the extra trailing commas then ;)

Reply:
Hopefully made life a bit easier for you and made the changes as suggestions that you can just apply in this PR directly